src/Microsoft.ML.TensorFlow/TensorFlowModel.cs

```diff
@@ -56,13 +56,6 @@ public DataViewSchema GetInputSchema()
         /// <param name="outputColumnName">The name of the requested model output.</param>
         /// <param name="addBatchDimensionInput">Add a batch dimension to the input e.g. input = [224, 224, 3] => [-1, 224, 224, 3].
         /// This parameter is used to deal with models that have unknown shape but the internal operators in the model require data to have batch dimension as well.</param>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]
```

Comment:
We don't have this sample?

Reply:
We didn't but I just modified an existing sample so that we can have a sample for this API :)